src/gurobi_optimods/opf/grbformulator.py

```diff
@@ -108,6 +108,13 @@ def lpformulator_optimize(alldata, model, opftype):
         for j in range(1, 1 + numbuses):
             bus = buses[j]
             evar[bus].PStart = 1.0
+        if alldata["branchswitching_mip"]:
+            zvar = alldata["MIP"]["zvar"]
+            branches = alldata["branches"]
+            numbranches = alldata["numbranches"]
+            for j in range(1, 1 + numbranches):
+                branch = branches[j]
+                zvar[branch].PStart = 1.0
```

Comment:
This can be done more cleanly in Python, without all the intermediate lookups:

```suggestion
        if alldata["branchswitching_mip"]:
            zvars = alldata["MIP"]["zvar"]
            for zvar in zvars.values():
                zvar.PStart = 1.0
```